### PR TITLE
Publish to NPM

### LINKED ISSUE #246: [FEATURE] Publish to NPM
## User Story
1. As a {Jayvee user}
2. I want {to use NPM as package registry}
3. So that {I don't need to get a GitHub PAT first when installing Jayvee}

## User Acceptance Criteria
- [ ] Packages are published to NPM

## Definitions of Done
- [ ] A PR has been opened and accepted
- [ ] All user acceptance criteria are met
- [ ] All tests are passing

### PR DESCRIPTION
**Pull request by georg-schwarz**
Closes #246.

### CODE REVIEW

File: apps/interpreter/package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@jvalue/interpreter",
+  "name": "@jvalue/jayvee",
```
Comment (georg-schwarz) [inline apps/interpreter/package.json:2]: Does it make sense to name the interpreter **the** jayvee package?

Comment (rhazn) [reply apps/interpreter/package.json:2]: I don't think so, should be jayvee-interpreter at least.

File: libs/monaco-editor/package.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@jvalue/monaco-editor",
+  "name": "@jvalue/jayvee-monaco",
```
Comment (georg-schwarz) [inline libs/monaco-editor/package.json:2]: Removed the `editor` postfix as it seems duplicate for me

**Review by rhazn — APPROVED** [review]

**Comment by rhazn** [comment]
Seems like some checks fail but otherwise looks okay to me 👍

**Pull request closed without merging**